.gitignore

```diff
@@ -132,4 +132,7 @@ dmypy.json
 .idea/
 
 #DS_Store
-*/.DS_Store
\ No newline at end of file
+*/.DS_Store
+
+# db
+*/*.db
```

Comment:
You can consider adding new line to end of file as it's a good practice. Same comment to other files too